Source/cmake/OptionsCommon.cmake

```diff
@@ -41,7 +41,9 @@ if (COMPILER_IS_GCC_OR_CLANG)
     set(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} -fno-exceptions -fno-strict-aliasing")
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c99")
     set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -fno-exceptions -fno-strict-aliasing -fno-rtti")
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++1y")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14 ")
```

Comment:
Both `-std=c++14` and `-Wno-expansion-to-defined` seem to be fine, I still have my GuruPlug at hand with 4.9.3 and it accepts the flags works:

```
~ % g++ -dumpversion
4.9.3
~ % g++ -std=c++14 -Wno-expansion-to-defined -o hi hi.cc
~ % file hi
hi: ELF 32-bit LSB executable, ARM, EABI5 version 1 (SYSV), dynamically linked, interpreter /lib/ld-linux.so.3, for GNU/Linux 2.6.32, BuildID[sha1]=1e3d82e1771096e637a741dc0bec3b669c6e850e, not stripped
~ %
```

JFTR, there's also an [upstream bug](https://bugs.webkit.org/show_bug.cgi?id=167643) with a patch to add `-Wno-expansion-to-defined` to the command line of GCC and Clang.